shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/runtime/ActionGenerator.java

```diff
@@ -26,6 +28,11 @@ public Stream<String> inputs() {
           return Stream.empty();
         }
 
+        @Override
+        public Lookup privateLookup() {
+          return MethodHandles.publicLookup();
```

Comment:
'Private lookup' returns a 'public lookup'?

Reply:
Yeah, poorly worded. I've renamed the method to `lookup` and added a clarification why this is public lookup in just this one instance.